src/twisted/web/http.py

```diff
@@ -1056,9 +1056,13 @@ def write(self, data):
             # if we don't have a content length, we send data in
             # chunked mode, so that we can support pipelining in
             # persistent connections.
+            # For no-body responses and CONNECT requests this
+            # does not apply and the header should not or must
```

Comment:
is it "*should* not" or "*must* not"?  seems like it should be one or the other.